tests/integration/construct_component_failures/go/main.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2016-2024, Pulumi Corporation.  All rights reserved.
```

Comment:
If this passes the `goheader` lint, I guess we don't lint test Go code?

Reply:
Apparently not :)  I made it consistent with the other go files now.